target/linux/ramips/image/mt7620.mk

```diff
@@ -878,6 +895,25 @@ define Device/tplink_archer-c20i
 endef
 TARGET_DEVICES += tplink_archer-c20i
 
+define Device/tplink_re200-v1
```

Comment:
please move the definition to the proper alphabetical position (behind Device/tplink_archer-mr200)

Reply:
Done